src/tests/JIT/Regression/JitBlue/Runtime_107146/Runtime_107146.cs

```diff
@@ -28,21 +28,19 @@ public class Runtime_107146
     [Fact]
     public static void TestEntryPoint()
     {
-        if (!Avx512BW.VL.IsSupported)
+        if (Avx512BW.VL.IsSupported) // Avx512BW implies Popcnt
```

Comment:
Where do we depend on the exact polarity of the branch?

Reply:
@tannergooding actually it seems that `DOTNET_EnablePOPCNT=0` doesn't turn `AVX512BW` so we do need to check `Popcnt.IsSupported` explicitly here

Reply:
> actually it seems that DOTNET_EnablePOPCNT=0 doesn't turn AVX512BW so we do need to check Popcnt.IsSupported explicitly here

👍. That's probably a missing implication we should fix in .NET 10 then.

`popcnt` is a bit different because for Intel it's formally part of `SSE4.2`, while AMD includes it as part of `SSE4a` (which is an AMD specific precursor to `SSE4.1` we don't expose).

It's identified by its own unique bit, but in effect you cannot have AVX or higher without it.

Reply:
(No need to fix it in this PR, I'll log an issue and track getting it resolved separately)